base/main.cpp

```diff
@@ -551,17 +553,17 @@ extern "C" int scummvm_main(int argc, const char * const argv[]) {
 			#ifndef FORCE_RTL
 			if (result.getCode() == Common::kNoError && !g_system->getEventManager()->shouldRTL())
 				break;
-			#endif
+#endif
```

Comment:
Now, the #endif does not match the indentation of the #ifndef. Thus, either have both unindented, or do not make this change at all.